app/models/game.rb

```diff
@@ -1,3 +1,34 @@
 class Game < ActiveRecord::Base
   scope :available, -> { where('black_player IS NULL OR white_player IS NULL') }
+  has_many :users
+  has_many :pieces
+
+  def populate_board!
```

Comment:
Instead of increasing the method length in Rubocop, we can break up this long method by having it call several small methods (say one to set up the bishops, one to setup the knights, etc. or have 1 method to set up all the pieces on a side and then pass in the color as a parameter)